extensions/pkg/controller/worker/templates/machineclasses.tpl.yaml

```diff
@@ -14,6 +14,8 @@ spec:
     listKind: MachineClassList
     plural: machineclasses
     singular: machineclass
+    shortNames:
+      - machcls
```

Comment:
Why not `machcl`? It is not consistent with the other short names that don't have `s` suffix.

Reply:
I thought it makes more sense to use `cls` instead of `cl` for `class`. Also in the previous versions of gardener before these short names were removed, it was `machcls` so I thought it should be fine.

Reply:
wdyt?

Reply:
👍

Reply:
I would rather suggest using `machclass` instead of `machcls`? We are not saving much by omitting 2 letters.

Reply:
done.